runtime/ftplugin/dart.lua

```diff
@@ -0,0 +1,3 @@
+vim.bo.commentstring = '// %s'
+
+vim.b.undo_ftplugin = vim.b.undo_ftplugin .. ' | setl commentstring<'
```

Comment:
`runtime/ftplugin/dart.vim` already sets this.

Reply:
True, must've missed this while reviewing the vimscript files, I've removed `dart.lua`